backend/job_board/admin.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+
+from .models import JobListing
+
+
+@admin.register(JobListing)
+class PostAdmin(admin.ModelAdmin):
+    model = JobListing
+    list_display = ("title", "company")
```

Comment:
can we add some filters and searchers?

Reply:
there's not going to be a lot of jobs :)